test/specs/modules/Dimmer/Dimmer-test.js

```diff
@@ -26,6 +26,13 @@ describe('Dimmer', () => {
       shallow(<Dimmer content={text} />)
         .should.contain.text(text)
     })
+
+    it('renders clickable buttons', () => {
```

Comment:
This test is already covered by `common.rendersChildren()` [above](https://github.com/Semantic-Org/Semantic-UI-React/pull/782/files#diff-65fcf9237dea10f9c70c3c54c9a8428eR14).  We can remove this one.